lib/mayaUsd/ufe/UsdUndoConnectionCommands.cpp

```diff
@@ -284,6 +284,104 @@ UsdUndoDeleteConnectionCommand::Ptr UsdUndoDeleteConnectionCommand::create(
     return std::make_shared<UsdUndoDeleteConnectionCommand>(srcAttr, dstAttr);
 }
 
+bool canRemoveSrcProperty(const PXR_NS::UsdAttribute& srcAttr)
```

Comment:
I think this helper function (and the one below) should be in the unnamed namespace near the top of this file (with the other helper functions).

Reply:
Could I move them to `mayaUsd/ufe/Utils.h`? I need them also for other issues I'm working on.

Reply:
Yes that is a good idea.

Reply:
I moved also the `isConnected` function since it is used by `canRemoveSrcProperty` and `canRemoveDstProperty`, I also need it for the issues I'm working on.